pandas/core/internals/managers.py

```diff
@@ -807,7 +812,7 @@ def _interleave(self):
         # TODO: https://github.com/pandas-dev/pandas/issues/22791
         # Give EAs some input on what happens here. Sparse needs this.
         if is_sparse(dtype):
-            dtype = dtype.subtype
+            dtype = dtype.subtype  # type: ignore
```

Comment:
according to the docstring, is_sparse is to check arrays. so isinstance(dtype, SparseDtype) could be used here to avoid the ignore?